roles/ceilometer-common/templates/etc/ceilometer/ceilometer.conf

```diff
@@ -53,13 +53,15 @@ host = 0.0.0.0
 port = {{ endpoints.ceilometer.port.backend_api }}
 
 [keystone_authtoken]
-identity_uri = {{ endpoints.identity_uri }}
-auth_uri = {{ endpoints.auth_uri }}
-admin_tenant_name = service
-admin_user = ceilometer
-admin_password = {{ secrets.service_password }}
+auth_url = {{ endpoints.keystone.url.admin }}
```

Comment:
Hard to tell, because that gets handed off to keystone auth middleware which does its own parsing of the data, and has it's own settings. I could try flipping it back, but it seems to work now, and matches what we do in nova and other services that use the same config block.